x-pack/plugins/security_solution/public/management/pages/trusted_apps/store/builders.ts

```diff
@@ -48,6 +48,7 @@ export const initialTrustedAppsPageState = (): TrustedAppsListPageState => ({
   },
   deletionDialog: initialDeletionDialogState(),
   creationDialog: initialCreationDialogState(),
+  policies: { type: 'UninitialisedResourceState' },
```

Comment:
What are your thoughts on putting this in creationDialog state property?

Reply:
I can do if that's preferable.
I placed outside so that the same set of code that retrieve it and sets it in the store is the same logic regardless if its the Crate use case, or the near future Edit use case.

Reply:
I guess you are right, it might be necessary for more places.